src/main/webapp/app/overview/course-overview.component.ts

```
@@ -124,6 +125,7 @@
     isExamStarted = false;
     private examStartedSubscription: Subscription;
     readonly MIN_DISPLAYED_COURSES: number = 6;
+    isLti: boolean = false;
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)